app/serializers/api/v1/stockit_item_serializer.rb

```diff
@@ -12,6 +12,7 @@ class StockitItemSerializer < ApplicationSerializer
     has_many :package_actions, serializer: PackageActionsSerializer, root: :item_actions
     has_one :storage_type, serializer: StorageTypeSerializer
     has_one :package_set, serializer: PackageSetSerializer::StockFormat
+    has_many :versions, serializer: VersionSerializer
```

Comment:
@abulsayyad123 my concern here is that this will make the serializer quite slow. Have you profiled this at all?

Reply:
StockitItemSerializer is used on PackagesController#create, search_stockit_items, remove_from_set, contained_packages, parent_containers, update, split_package, move, designate, register_quantity_change

These are all places we don't need version history right? Particularly `search_stockit_items`

cc @patrixr

Reply:
Suggestion from @patrixr : Create an endpoint `/api/v1/packages/1/versions` and have it return all versions for the package